patches/api/0441-Fix-PlayerSwapHandItemsEvent-throwing-exception-when.patch

```diff
@@ -0,0 +1,29 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Tamion <70228790+notTamion@users.noreply.github.com>
+Date: Mon, 25 Sep 2023 19:55:51 +0200
+Subject: [PATCH] Fix PlayerSwapHandItemsEvent throwing exception when mainhand
+ or offhand set to null
+
+
+diff --git a/src/main/java/org/bukkit/event/player/PlayerSwapHandItemsEvent.java b/src/main/java/org/bukkit/event/player/PlayerSwapHandItemsEvent.java
+index 9f592317c920589c22a5fb8e916c6ca58ebe5c59..8ec2d1689399f9f89684ca4fe145d7553a72dff4 100644
+--- a/src/main/java/org/bukkit/event/player/PlayerSwapHandItemsEvent.java
++++ b/src/main/java/org/bukkit/event/player/PlayerSwapHandItemsEvent.java
+@@ -42,7 +42,7 @@ public class PlayerSwapHandItemsEvent extends PlayerEvent implements Cancellable
+      * @param mainHandItem new item in the main hand
+      */
+     public void setMainHandItem(@Nullable ItemStack mainHandItem) {
+-        this.mainHandItem = mainHandItem;
++        this.mainHandItem = mainHandItem == null ? new ItemStack(org.bukkit.Material.AIR) : mainHandItem; // Paper
```

Comment:
Use `ItemStack#empty()` instead of a new stack of Air so the amount is set to 0. Same for below.

Reply:
Thanks for the improvements